db/schema.rb

```diff
@@ -35,6 +38,17 @@
     t.index ['post_id'], name: 'index_likes_on_post_id'
   end
 
+  create_table 'posts', force: :cascade do |t|
```

Comment:
- [ ] You are required to create all tables for `schema.rb` using migrations. I only see one migration file in your codebase. For example, you can create one for the users table by entering something similar to this in the terminal.
`rails g migration Users name:string photo:string posts_counter:integer`